### ISSUE TITLE
marineanalvar fails with "soca_fields::get():  cannot find field eastward_sea_water_velocity"

### ISSUE THREAD

**Issue by RussTreadon-NOAA** [NOAA-EMC/GDASApp#1421]
GDASApp based g-w CI for C48mx500_3DVarAOWCDA and C48mx500_hybAOWCDA aborts when running gdas_marineanlvar for 20210325 00Z with the error message
```
0: OOPS_STATS GMRESR end                               - Runtime:     8.53 sec,  Local Memory:   442.61 Mb
 0: soca_fields::get():  cannot find field eastward_sea_water_velocity
 2: soca_fields::get():  cannot find field eastward_sea_water_velocity
 3: soca_fields::get():  cannot find field eastward_sea_water_velocity
 4: soca_fields::get():  cannot find field eastward_sea_water_velocity
 5: soca_fields::get():  cannot find field eastward_sea_water_velocity
 6: soca_fields::get():  cannot find field eastward_sea_water_velocity
 7: soca_fields::get():  cannot find field eastward_sea_water_velocity
 8: soca_fields::get():  cannot find field eastward_sea_water_velocity
 9: soca_fields::get():  cannot find field eastward_sea_water_velocity
10: soca_fields::get():  cannot find field eastward_sea_water_velocity
11: soca_fields::get():  cannot find field eastward_sea_water_velocity
12: soca_fields::get():  cannot find field eastward_sea_water_velocity
13: soca_fields::get():  cannot find field eastward_sea_water_velocity
14: soca_fields::get():  cannot find field eastward_sea_water_velocity
15: soca_fields::get():  cannot find field eastward_sea_water_velocity
 0: OOPS Ending   2024-12-18 01:45:50 (UTC+0000)
 0: Abort(1) on node 0 (rank 0 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
 1: soca_fields::get():  cannot find field eastward_sea_water_velocity
 1: Abort(1) on node 1 (rank 1 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 1) - process 1
 2: Abort(1) on node 2 (rank 2 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 1) - process 2
```

The input yaml, `var.yaml`, includes `eastward_sea_water_velocity` as an analysis variable
```
  analysis variables:
  - sea_ice_area_fraction
  - sea_ice_thickness
  - sea_ice_snow_thickness
  - sea_water_salinity
  - sea_water_potential_temperature
  - eastward_sea_water_velocity
  - northward_sea_water_velocity
  - sea_surface_height_above_geoid
```
but does not list this or related variables in the background state variables
```
  background:
    read_from_file: 1
    basename: ./INPUT/
    ocn_filename: MOM.res.nc
    ice_filename: cice.res.nc
    date: '2021-03-24T21:00:00Z'
    state variables:
    - sea_ice_area_fraction
    - sea_ice_thickness
    - sea_ice_snow_thickness
    - snow_ice_surface_temperature
    - air_temperature
    - bulk_ice_salinity
    - sea_water_salinity
    - sea_water_potential_temperature
    - sea_surface_height_above_geoid
    - sea_water_cell_thickness
    - ocean_mixed_layer_thickness
    - sea_water_depth
```

jcb-gdas PR [#58](https://github.com/NOAA-EMC/jcb-gdas/pull/58) commented out entries for the eastward and northward water velocities in [`model/marine/marine_background.yaml.j2`](https://github.com/NOAA-EMC/jcb-gdas/pull/58/files#diff-1e2832c4e6daa169e0fcec932babcd3e34e6cbb0712b93b55169d04c9867a0fc)
```
#- eastward_sea_water_velocity
#- northward_sea_water_velocity
```

However, GDASApp `develop` [`parm/soca/marine-jcb-base.yaml`](https://github.com/NOAA-EMC/GDASApp/blob/develop/parm/soca/marine-jcb-base.yaml) still includes these fields as analysis variables on[ line 127](https://github.com/NOAA-EMC/GDASApp/blob/develop/parm/soca/marine-jcb-base.yaml#L27)

Removing 
```
- eastward_sea_water_velocity
- northward_sea_water_velocity
```
from `parm/soca/marine-jcb-base.yaml` and rerunning from gdas_marineanlinit allows gdas_marineanlvar to successfully complete. I do not know, however, if downstream jobs will run OK. 

Is removing the water velocity variables from the analysis variables section of `parm/soca/marine-jcb-base.yaml` the right solution?
```
@@ -24,7 +24,7 @@ minimizer: RPCG
 final_diagnostics_departures: oman
 final_prints_frequency: PT3H
 number_of_outer_loops: 1
-analysis_variables: [sea_ice_area_fraction, sea_ice_thickness, sea_ice_snow_thickness, sea_water_salinity, sea_water_potential_t
emperature, eastward_sea_water_velocity, northward_sea_water_velocity, sea_surface_height_above_geoid]
+analysis_variables: [sea_ice_area_fraction, sea_ice_thickness, sea_ice_snow_thickness, sea_water_salinity, sea_water_potential_t
emperature, sea_surface_height_above_geoid]

 # Model things
 # ------------
```

**Comment by RussTreadon-NOAA** [comment]
Attention @guillaumevernieres , @AndrewEichmann-NOAA , @JohnSteffen-NOAA

**Comment by guillaumevernieres** [comment]
@RussTreadon-NOAA , this PR https://github.com/NOAA-EMC/GDASApp/pull/1417 should fix this issue.

**Comment by RussTreadon-NOAA** [comment]
Thank you @guillaumevernieres for letting me know a fix is in the works.  Hopefully PR [#1417](https://github.com/NOAA-EMC/GDASApp/pull/1417) can be merged into GDASApp `develop` soon.  `stable-nightly` will remain broken until this fix is in.

**Comment by guillaumevernieres** [comment]
working on it right now @RussTreadon-NOAA .

**Comment by guillaumevernieres** [comment]
fixed with #1417
